packages/web/src/components/gcds-link/stories/overview.mdx

```diff
@@ -0,0 +1,63 @@
+import { Meta, Canvas, Story } from '@storybook/blocks';
+import * as Link from './gcds-link.stories';
+
+<Meta of={Link} name="Overview" />
+
+# Link<br/>`<gcds-link>`
+
+_Also called: call to action, CTA, link._
+
+A Link is an interactive object that highlights an important or common action.
```

Comment:
This seems to still be using the button content. Can we change this to:
`A navigational element that allows users to navigate to a new page, website, or section on the current page.`